onnxruntime/core/automl/featurizers/src/FeaturizerPrep/Featurizer.h

```diff
@@ -0,0 +1,163 @@
+// ----------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License
+// ----------------------------------------------------------------------
+#pragma once
+
+#include <memory>
+#include <tuple>
+
+namespace Microsoft {
+namespace Featurizer {
```

Comment:
Featurizer sounds too specific for a namespace. Maybe **AutoML** instead?

Reply:
This is not our code. AutoML team. This code will be shared with them but hosted by ORT.

---
In reply to: [311298415](https://github.com/microsoft/onnxruntime/pull/1573#discussion_r311298415) [](ancestors = 311298415)